app/models/translation.rb

```diff
@@ -10,6 +10,20 @@ class Translation < ActiveRecord::Base
 
   @@root_folder_name = 'public/srt/'
 
+
+  @@STATUS_HASH = {0 => :incomplete, 1 => :complete, 2 => :complete_with_priority}
+  @@POINTS_HASH = {:incomplete => 0, :complete => 1, :complete_with_priority => 2}
```

Comment:
You should probably insert a comment here that these hashes aren't just the reverse of each other. It can be a little ambiguous.